config/config.go

```diff
@@ -299,6 +299,9 @@ type Cluster struct {
 type Subnet struct {
 	AvailabilityZone  string `yaml:"availabilityZone,omitempty"`
 	InstanceCIDR      string `yaml:"instanceCIDR,omitempty"`
+	SubnetId          string `yaml:"subnetId,omitempty"`
+	Create            bool
```

Comment:
this looks redundant, it is always `== (subnet.SubnetId != "")` so might as well remove it